### ISSUE TITLE
 /usr/bin/env: ‘node --no-deprecation’: No such file or directory

### ISSUE THREAD

**Issue by vporton** [aragon/aragon-react-boilerplate#92]
```
$ npm start

> placeholder-app-name@1.0.0 start /home/porton/t/aragon-react-boilerplate
> npm run start:ipfs


> placeholder-app-name@1.0.0 start:ipfs /home/porton/t/aragon-react-boilerplate
> aragon run --files dist

/usr/bin/env: ‘node --no-deprecation’: No such file or directory
/usr/bin/env: use -[v]S to pass options in shebang lines
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! placeholder-app-name@1.0.0 start:ipfs: `aragon run --files dist`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the placeholder-app-name@1.0.0 start:ipfs script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/porton/.npm/_logs/2019-12-22T20_23_49_509Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! placeholder-app-name@1.0.0 start: `npm run start:ipfs`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the placeholder-app-name@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/porton/.npm/_logs/2019-12-22T20_23_49_518Z-debug.lo
```

**Comment by sohkai** [comment]
@vporton I believe this should be fixed in the latest version of the CLI (v7.0.2), but if not we should continue looking into this @0xGabi.

**Comment by 0xGabi** [comment]
This was fixed on latest patch version (v7.0.3)
